dbt/adapters/databricks/connections.py

```diff
@@ -477,16 +478,15 @@ class DatabricksConnectionManager(SparkConnectionManager):
     credentials_provider: Optional[TCredentialProvider] = None
     _user_agent = f"dbt-databricks/{__version__}"
 
+    def __init__(self, profile: AdapterRequiredConfig, mp_context: SpawnContext) -> None:
```

Comment:
Can remove.  I was initially going to use the initializer to make the client, and reuse the client for making pipeline calls, but the pipeline API was complicated enough that I want to hold off until we can replace with an SDK client, especially because the 'open' method is a class method and can't reference a client associated with an instance.